antiplag/views.py

```diff
@@ -63,10 +78,10 @@ def post(self, request):
                 type=Document.DocumentType.TEXT,
                 text_raw=text_raw,
             )
-
+        
         # Run background task that processes documents
-        process_documents.delay(submission.id)
-
+        process_documents(submission.id)
```

Comment:
Please, revert the change to async task after you finished the synchronous debugging. We cannot merge this into production.